packages/server/utils/authorization.ts

```diff
@@ -62,3 +62,18 @@ export const isUserInOrg = async (userId: string, orgId: string, dataLoader: Dat
     .load({userId, orgId})
   return !!organizationUser
 }
+
+export const isTeamMemberOrOrgAdmin = async (
+  authToken: AuthToken,
+  teamId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const viewerId = getUserId(authToken)
+  // First check if they're a team member (faster check)
+  if (isTeamMember(authToken, teamId)) return true
+
+  // If not, check if they're an org admin
+  const team = await dataLoader.get('teams').load(teamId)
+  if (!team) return false
+  return isUserOrgAdmin(viewerId, team.orgId, dataLoader)
+}
```

Comment:
(for future reference) This diff has the right intent, but we could have used the existing `isUserOrgAdmin` function and avoid adding a new function:

https://github.com/ParabolInc/parabol/blob/abce357358353cd4844d99e349177b1bc38ab4fc/packages/server/utils/authorization.ts#L50-L57

 This would have been better because we would have kept the diffs minimal.